views/agendas.tmpl

```diff
@@ -253,7 +253,7 @@
                 <tbody>
                     <td><a href="/agenda/{{.ID}}">{{.ID}}</a></td>
                     <td class="d-none d-md-table-cell truncate">{{.Description}}</td>
-                    <td class="text-right">{{.Status}}</td>
```

Comment:
This one is non-numeric.  Just looks better?

Reply:
Yes, IMO